freqtrade/persistence.py

```diff
@@ -266,6 +266,7 @@ def update(self, order: Dict) -> None:
             logger.info('%s_SELL has been fulfilled for %s.', order_type.upper(), self)
         elif order_type == 'stop_loss_limit':
             self.stoploss_order_id = None
+            self.close_rate_requested = self.stop_loss
```

Comment:
this is not entirely true. the order is fulfilled somewhere between 'limit' and 'price' . ideally we should use order['average']

Reply:
this is the "requested" rate - not the "close_rate"... the "close_rate_requested" should be the "real" stoploss value - unless i miss something?

For real orders, it's set as "limit" we request ... that also doesn't guarantee you it's fulfilled at that price, it's that price or better.